### ISSUE TITLE
Add support for title

### ISSUE THREAD

**Issue by KeithCu** [wommy/keith_tmst#2]
Hi;

Can you add support for the title? I can make it customizable via jinja but I need help with the HTML. Thanks!!

**Comment by wommy** [comment]
what else needs variable work?

in a second I'm going to create a list

**Comment by KeithCu** [comment]
I already made variables for everything I care about and can do it for this also. I just want any title to appear and look good.

**Comment by wommy** [comment]
yeah I get that,
what's your list so I can add em to my end

**Comment by KeithCu** [comment]
```
<body>
	<div style="container">
	<header>
		<img src="static/images/{{background_image}}" alt="">
	</header>
	<main class="max-w-prose flex flex-col align-center mx-auto text-center py-16">
		<h2 class="font-medium my-0">How to purchase this Domain Name</h2>
		<p class="text-lg">
			<strong>{{domain_friendly}}</strong> is a Premium Domain Name For Sale.  Inquires please send contact information with name, phone number best time to follow up and to answer any questions, regarding the purchase of this domain name and or,  you may submit a private bid for consideration to <a class="text-gray-600 hover:text-gray-800" href="gemartschl@aol.com">gemartschl@aol.com</a>
		</p>
		<a href="/" class="text-white bg-black bg-opacity-60 w-48 mx-auto py-4 no-underline rounded font-bold hover:bg-opacity-40">FIND OUT MORE</a>
	</main>

```

**Comment by KeithCu** [comment]
Thanks for your help. The page looks good otherwise...

**Comment by wommy** [comment]
so we got

- [x] background_image
- [x] domain_friendly

**Comment by wommy** [comment]
solved with b17e98d98725db357badea4ee102cfb3806d63dc
